user_guide/src/quickstart/intro.md

```diff
@@ -88,3 +88,137 @@ So the eager snippet above would become:
     .collect()
 )
 ```
+
+## Using this Cookbook
+
+If you're new to `Polars` we reccommend reading through this Cookbook in its entirity to properly familiarize yourself with the `DataFrame` library.
```

Comment:
Is this a recommendation for the whole book? Because If anything, I want users to read the expression guide. That should remain the focus point of the book.

I also don't want to make the distincition between the eager and lazy API's that much. Because the API' are almost the same. Both use the expression API.

So in this sentence: "A 5 minute walkthrough of basic `Polars` usage introducing you to the Eager API. |", maybe drop the `Eager` word, and just go for "API".

Reply:
> Is this a recommendation for the whole book? Because If anything, I want users to read the expression guide.

I only mentioned the whole book to start somewhere with the wording. I'd love to narrow that recommendation down. A big part of the motivation behind adding a walk-through is to direct attention to the places it should be directed, but having done so after providing a little bit exposure so that dots can be more easily connected.

What I wanted to do next was add a Fundamentals setup page where we can be like "when you used X to do Y you were using a Polars Expression. Mention other important related topics like contexts, etc.. Then lead into the next/first topic (expressions).

I'm not in love with the current state of the walk-through, but maybe we can continue to revise it to figure it out!

> don't distinguish between APIs

Ooo yes that is a much better way to approach it!

Btw! Don't feel forced to merge this PR. My goal is only to improve the docs. If we don't think this absolutely improves it then we can trash it and rethink it.